qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -224,6 +372,36 @@ def __init__(self, parent: Instrument, name: str, channel: str) -> None:
         self.add_parameter('fastsweep',
                            parameter_class=LuaSweepParameter)
 
+        self.add_parameter('timetrace_npts',
+                           initial_value=500,
+                           label='Number of points',
+                           get_cmd=None,
+                           set_cmd=None)
+
+        self.add_parameter('timetrace_dt',
+                           initial_value=1e-3,
+                           label='Time resolution',
+                           unit='s',
+                           get_cmd=None,
+                           set_cmd=None)
+
+        self.add_parameter(name='time_axis',
+                           label='Time',
+                           unit='s',
+                           vals=vals.Arrays(shape=(self.timetrace_npts,)),
+                           parameter_class=TimeAxis)
```

Comment:
The `time_axis` parameter should not be included in the snapshot as it doesn't give any new information not covered by other parameters and also is potentially huge. I think passing `snapshot_value=False` to `add_parameter` should do the trick.